catalystneuro/nwb-conversion-tools
# Test hdmf import for GenericDataChunkIterator

## Motivation
When [hdmf PR #672](https://github.com/hdmf-dev/hdmf/pull/672) gets merged and attached to a new release, this PR will update the requirements to that version of `hdmf`. We can setup a downstream testing pipeline ahead of time, though, to ensure it doesn't break anything prior to merge.

## How to test the behavior?
All interfaces that utilize inheritors of the `GenericDataChunkIterator` class will now import that from `hdmf` - our tests of all those interfaces will therefore check that nothing is awry with our calling of the slightly different upstream function.

## Checklist
